src/coreclr/src/vm/codeversion.h

```diff
@@ -293,6 +307,9 @@ class NativeCodeVersionNode
 #ifdef HAVE_GCCOVER
     PTR_GCCoverageInfo m_gcCover;
 #endif
+#ifdef FEATURE_ON_STACK_REPLACEMENT
+    CORINFO_OSR_INFO m_osrInfo;
```

Comment:
I tends to not work very well to use JIT/EE interface structures inside runtime datastructure with longer lifetime.